Sources/Subprocess/Configuration.swift

```diff
@@ -632,26 +632,7 @@ internal struct TrackedFileDescriptor: ~Copyable {
             return
         }
 
```

Comment:
there's still this `guard self.closeWhenDone` here which is probably not intended as we're never closing here (which is great)